### PR TITLE
Update references and readme

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request updates reference URLs from `https://all-rebilly.redoc.ly/` to `https://www.rebilly.com/catalog/all/`.

This pull request also updates [the readme](https://github.com/Rebilly/api-definitions/pull/1731/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) to point to `https://www.rebilly.com/catalog/all/`. 

## Checklist

- [X] Writing style
- [X] API design standards

### CODE REVIEW

**Comment by noon-dawg** [comment]
@Rebilly/php, @Rebilly/js, @Rebilly/docs please review

File: openapi/components/schemas/DepositStrategy.yaml [hunk 1]
```diff
@@ -74,7 +74,7 @@ properties:
       Custom amount restrictions.
       If this value is `null`,  custom amounts are prohibited.
       If `customAmount` is not specified when a deposit request is created, amount restrictions are determined from the chosen strategy.
-      For more information, see the [`strategyId` property](https://all-rebilly.redoc.ly/tag/Deposits/#operation/PostDepositRequest#!path=strategyId&t=request).
+      For more information, see the [`strategyId` property](/Deposits/PostDepositRequest#!t=request&path=strategyId).
```
Comment (ashkarpetin) [inline openapi/components/schemas/DepositStrategy.yaml:77]: Seems wrong link

File: openapi/components/parameters/collectionExpand.yaml [hunk 2]
```diff
@@ -5,6 +5,6 @@ description: |-
   property of the response. This field accepts a comma-separated list of objects.
 
   For more information, see
-  [Expand to include embedded objects](https://all-rebilly.redoc.ly/#section/Expand-to-include-embedded-objects).
+  [Expand to include embedded objects](/#section/Expand-to-include-embedded-objects).
```
[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/parameters/collectionExpand.yaml]: Thanks @ashkarpetin 🙏 

I updated relative links to absolute, and fixed a few more broken links in [20b382e](https://github.com/Rebilly/api-definitions/pull/1731/commits/20b382ef4ea68a0ec398b0a94e1bade1821458db)

**Review by ashkarpetin — APPROVED** [review]

**Comment by noon-dawg** [comment]
@Rebilly/php please review

**Review by nanobash — APPROVED** [review]

**Pull request merged**